src/lavinmq/log_formatter.cr

```diff
@@ -3,13 +3,57 @@ require "log"
 module LavinMQ
   struct JournalLogFormat < ::Log::StaticFormatter
     def run
+      severity
       source
       context(before: '[', after: ']')
       data(before: '[', after: ']')
       string ' '
       message
       exception
     end
+
+    private def severity : Nil
+      @io << '<' << severity_to_priority << '>' << ' '
+    end
+
+    private def exception(*, before = '\n', after = nil) : Nil
+      if ex = @entry.exception
+        @io << '\n'
+        inspect_with_backtrace(ex)
+        @io << after
+      end
+    end
+
+    private def inspect_with_backtrace(ex : Exception) : Nil
+      severity
+      @io << message << " (" << ex.class << ")\n"
+
+      ex.backtrace?.try &.each do |frame|
+        severity
+        @io.print "  from "
+        @io.puts frame
+      end
+
+      if cause = ex.cause
+        severity
+        @io << "Caused by: "
+        inspect_with_backtrace(cause)
+      end
+
+      @io.flush
+    end
+
+    private def severity_to_priority
+      case @entry.severity
+      in ::Log::Severity::Trace  then 7 # journald doesn't have trace
+      in ::Log::Severity::Debug  then 7
+      in ::Log::Severity::Info   then 6
+      in ::Log::Severity::Notice then 5
+      in ::Log::Severity::Warn   then 4
+      in ::Log::Severity::Error  then 3
+      in ::Log::Severity::Fatal  then 2
```

Comment:
TIL `in`, great! 

Looks like we need `None` as well 

https://github.com/crystal-lang/crystal/blob/7b9e2ef80f52d5b5bd2627072ba31619c5ea57a4/src/log/entry.cr#L20-L32

```
#37 28.96 In src/lavinmq/log_formatter.cr:47:7
#37 28.96 
#37 28.96  47 | case @entry.severity
#37 28.96       ^
#37 28.96 Error: case is not exhaustive for enum Log::Severity.
```